Dockerfile

```diff
@@ -16,13 +16,15 @@ COPY controllers/ controllers/
 COPY converters/ converters/
 COPY constants/ constants/
 COPY builder/ builder/
+COPY utils/ utils/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+#FROM gcr.io/distroless/static:nonroot
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
there is now the ubi-micro, maybe its worth to try to use that and see how it goes :)

Reply:
@spolti with minimal the operator image is 150 mb with micro the operator image 82.8 mb , switching to micro